src/module/components/ngx-auth-firebaseui/auth.component.ts

```diff
@@ -163,6 +163,12 @@ export class AuthComponent implements OnInit, AfterViewInit, OnChanges, OnDestro
     if (changes.messageOnAuthSuccess || changes.messageOnAuthError) {
       this.updateAuthSnackbarMessages();
     }
+    if (changes.min) {
+      this.min = this.min != null ? Math.max(this.min, this.config.passwordMinLength) : this.config.passwordMinLength;
+    }
+    if (changes.max) {
+      this.max = this.max != null ? Math.min(this.max, this.config.passwordMaxLength) : this.config.passwordMaxLength;
```

Comment:
I assume that the logic is wrong

`this.min = this.min != null ? Math.max(this.min, this.config.passwordMinLength) : this.config.passwordMinLength;`

in the above part, if `this.min` is not null and `this.min` is bigger than `this.config.passwordMinLength`
then you will pick the wrong value which is the one configured here `this.config.passwordMinLength`

Do you agree with me ?

`this.max = this.max != null ? Math.min(this.max, this.config.passwordMaxLength) : this.config.passwordMaxLength;
`

the same thing here above: 

if `this.max` is smaller than the one in the configuration, then you will pick the wrong value... 


see more below:

```
 max(...values: number[]): number;
    /**
      * Returns the smaller of a set of supplied numeric expressions.
      * @param values Numeric expressions to be evaluated.
      */
    min(...values: number[]): number;
    /**
      * Returns the value of a base expression taken to a specified power.
      * @param x The base value of the expression.
      * @param y The exponent value of the expression.
      */
```

Reply:
I don't think so. 

> The Math.max() function returns the largest of zero or more numbers.

If `this.min` is `not null` and `>` to `this.config.passwordMinLength`, `Math.min` will return `this.min`.